src/python/pants/backend/terraform/fmt.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
We want this to be in `backend/terraform/lint/fmt.py`, and then `tfmt` and `validate` in their own subdirectories. Ack that that's lots of boilerplate - but we want it so that linters/formatters are _always_ explicitly opted into imo. 

(We had a lot of issues in V1 land that we automatically opted you in to a lot of functionality that was hard to disable)

Reply:
Source has been reorganized as suggested.